src/scenes/home/press/civicXBadge/civicXBadge.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import civicX from '../../../../images/civic-x.png';
+import styles from './civicXBadge.css';
+
+const CivicXBadge = () => (
+  <div>
+    <a href="http://cvcx.org/veterans-solutions-lab/">
```

Comment:
Can you use the outboundLink component here? I should've caught this earlier, I apologize.

Globally search for usages of `<OutboundLink` to see example implementations.

Once done, I'll approve and merge. Good work again.

Reply:
No worries! Will push that in a minute

Reply:
@kylemh Looks like there's an analyticsEventLabel prop on that component, would something like `CivicX Accelerator Badge` be descriptive enough for that?

Reply:
@Jacfem perf 💯

Reply:
@kylemh cool, updated!  I used `featuredJob.js`'s implementation of it as inspiration